website/docs/admin_server_market.md

```diff
@@ -0,0 +1,44 @@
+---
+id: admin_server_market
+title: AYON Market
+sidebar_label: AYON Market
+description: Downloading addons and releases from the market.
+---
+
+:::info
+To access the market, you need to use AYON Cloud or have a local instance connected to Ynput Cloud.
```

Comment:
What do you think about extending the note with info about connecting to Ynput Cloud, or maybe just a screenshot about connection? 
![image](https://github.com/user-attachments/assets/63aa5244-fdcb-4737-90f9-01beeb33161c)

Reply:
Added.